scripts/queries.py

```diff
@@ -0,0 +1,108 @@
+queries = {
+    "content_model_distribution": """
+        SELECT ?model (COUNT(?obj) as ?count)
+        FROM <#ri>
+        WHERE {
+            ?obj <info:fedora/fedora-system:def/model#hasModel> ?model;
+        }
+        GROUP BY ?model
+    """,
+
+    "object_count": """
+        SELECT (COUNT(?obj) as ?count)
+        FROM <#ri>
+        WHERE {
+            ?obj <info:fedora/fedora-system:def/model#hasModel> <info:fedora/fedora-system:FedoraObject-3.0> .
+        }
+        LIMIT 1
+    """,
+
+    "active_deleted_count": """
```

Comment:
Should this also get the count of objects with the 'inactive' state `<info:fedora/fedora-system:def/model#Inactive>`?